examples/inference/api_server_simple/query_single.py

```diff
@@ -75,8 +75,35 @@
     json=sample_input,
     stream=args.streaming_response,
 )
+try:
```

Comment:
I don't' think we can add the debug info in the examples since they are for users.  Need to figure out a way to print this on the serve side.